<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>moux</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QFaktury</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/qfaktury.qrc">
    <normaloff>:/res/icons/qfaktury_48.png</normaloff>:/res/icons/qfaktury_48.png</iconset>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidgetMain">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabInvoices">
       <attribute name="title">
        <string>Faktury</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelDisplayFrom">
            <property name="text">
             <string>Wyświetlaj dokumenty od:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEditFilterStart">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
            <property name="correctionMode">
             <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
            </property>
            <property name="displayFormat">
             <string>dd/MM/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
            <property name="date">
             <date>
              <year>2009</year>
              <month>3</month>
              <day>1</day>
             </date>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelDisplayTo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>40</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>do:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEditFilterEnd">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
            <property name="wrapping">
             <bool>true</bool>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="displayFormat">
             <string>dd/MM/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
            <property name="date">
             <date>
              <year>2010</year>
              <month>1</month>
              <day>1</day>
             </date>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonApply">
            <property name="text">
             <string>Zastosuj</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>180</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="tableViewInvoices">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>true</bool>
          </property>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabGoods">
       <attribute name="title">
        <string>Towary/Usługi</string>
       </attribute>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QTableView" name="tableViewCommodities">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCounterparties">
       <attribute name="title">
        <string>Kontrahenci</string>
       </attribute>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QTableView" name="tableViewCounterparties">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBarEditor">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>&amp;Program</string>
    </property>
    <addaction name="actionProgram_CompanyInfo"/>
    <addaction name="actionProgram_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionProgram_ExportToEDI"/>
    <addaction name="actionProgram_ExportToPDF"/>
    <addaction name="actionExportToXML"/>
    <addaction name="actionProgram_ImportEDI"/>
    <addaction name="actionProgram_ImportXML"/>
    <addaction name="actionProgram_Exit"/>
   </widget>
   <widget class="QMenu" name="menuInvoices">
    <property name="title">
     <string>&amp;Faktury</string>
    </property>
    <addaction name="actionInvoices_InvoiceNew"/>
    <addaction name="actionInvoices_InvoiceGross"/>
    <addaction name="actionInvoices_Bill"/>
    <addaction name="separator"/>
    <addaction name="actionInvoices_InvoiceProForma"/>
    <addaction name="actionInvoices_InvoiceCorrective"/>
    <addaction name="actionInvoices_InvoiceDuplicate"/>
    <addaction name="separator"/>
    <addaction name="actionInvoices_Remove"/>
    <addaction name="actionInvoices_Edit"/>
   </widget>
   <widget class="QMenu" name="menuCounterparties">
    <property name="title">
     <string>&amp;Kontrahenci</string>
    </property>
    <addaction name="actionCounterparties_Add"/>
    <addaction name="actionCounterparties_Remove"/>
    <addaction name="actionCounterparties_Edit"/>
   </widget>
   <widget class="QMenu" name="menuCommodities">
    <property name="title">
     <string>&amp;Towary</string>
    </property>
    <addaction name="actionCommodities_Add"/>
    <addaction name="actionCommodities_Remove"/>
    <addaction name="actionCommodities_Edit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>P&amp;omoc</string>
    </property>
    <addaction name="actionHelp_Help"/>
    <addaction name="actionHelp_BugReport"/>
    <addaction name="separator"/>
    <addaction name="actionHelp_AboutQt"/>
    <addaction name="actionHelp_About"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>Dodatk&amp;i</string>
    </property>
   </widget>
   <addaction name="menuProgram"/>
   <addaction name="menuInvoices"/>
   <addaction name="menuCounterparties"/>
   <addaction name="menuCommodities"/>
   <addaction name="menuPlugins"/>
   <addaction name="menuHelp"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionProgram_CompanyInfo"/>
   <addaction name="separator"/>
   <addaction name="actionInvoices_InvoiceNew"/>
   <addaction name="actionInvoices_InvoiceProForma"/>
   <addaction name="separator"/>
   <addaction name="actionCounterparties_Add"/>
   <addaction name="separator"/>
   <addaction name="actionCommodities_Add"/>
   <addaction name="separator"/>
   <addaction name="actionProgram_Settings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInvoices_Remove">
   <property name="icon">
    <iconset resource="../../res/qfaktury.qrc">
     <normaloff>:/res/icons/usun_fakture.png</normaloff>:/res/icons/usun_fakture.png</iconset>
   </property>
   <property name="text">
    <string>Usuń</string>
   </property>
   <property name="iconText">
    <string>Usuń</string>
   </property>
   <property name="shortcut">
    <string>U</string>
   </property>
   <property name="name" stdset="0">
    <string>fakturyUsunAction</string>
   </property>
  </action>
  <action name="actionInvoices_InvoiceNew">
   <property name="icon">
    <iconset resource="../../res/qfaktury.qrc">
     <normaloff>:/res/icons/nowa_faktura.png</normaloff>:/res/icons/nowa_faktura.png</iconset>
   </property>
   <property name="text">
    <string>Faktura VAT</string>
   </property>
   <property name="iconText">
    <string>Nowa faktura</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
   <property name="name" stdset="0">
    <string>fakturyDodajAction</string>
   </property>
  </action>
  <action name="actionCounterparties_Add">
   <property name="icon">
    <iconset resource="../../res/qfaktury.qrc">
     <normaloff>:/res/icons/dodaj_kontrahentow.png</normaloff>:/res/icons/dodaj_kontrahentow.png</iconset>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
   <property name="iconText">
    <string>Kontrahent</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
   <property name="name" stdset="0">
    <string>kontrahenciDodajAction</string>
   </property>
  </action>
  <action name="actionCounterparties_Remove">
   <property name="icon">
    <iconset resource="../../res/qfaktury.qrc">
     <normaloff>:/res/icons/usun_kontrahentow.png</normaloff>:/res/icons/usun_kontrahentow.png</iconset>
   </property>
   <property name="text">
    <string>Usuń</string>
   </property>
   <property name="iconText">
    <string>Usuń</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
   <property name="name" stdset="0">
    <string>kontrahenciUsunAction</string>
   </property>
  </action>
  <action name="actionCounterparties_Edit">
   <property name="icon">
    <iconset resource="../../res/qfaktury.qrc">
     <normaloff>:/res/icons/edytuj_kontrahentow.png</normaloff>:/res/icons/edytuj_kontrahentow.png</iconset>
   </property>
   <property name="text">
    <string>Edycja</string>
   </property>
   <property name="iconText">
    <string>Edycja</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="name" stdset="0">
    <string>kontrahenciEdycjaAction</string>
   </property>
  </action>
  <action name="actionProgram_Exit">
   <property name="icon">
    <iconset resource="../../res/qfaktury.qrc">
     <normaloff>:/res/icons/koniec.png</normaloff>:/res/icons/koniec.png</iconset>
   </property>
   <property name="text">
    <string>Koniec</string>
   </property>
   <property name="iconText">
    <string>Koniec</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
   <property name="name" stdset="0">
    <string>plikKoniecAction</string>
   </property>
  </action>
  <action name="actionInvoices_Edit">
   <property name="icon">
    <iconset resource="../../res/qfaktury.qrc">
     <normaloff>:/res/icons/edytuj_fakture.png</normaloff>:/res/icons/edytuj_fakture.png</iconset>
   </property>
   <property name="text">
    <string>Edytuj</string>
   </property>
   <property name="iconText">
    <string>Edytuj</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
   <property name="name" stdset="0">
    <string>fakturyEdAction</string>
   </property>
  </action>
  <action name="actionProgram_CompanyInfo">
   <property name="icon">
    <iconset resource="../../res/qfaktury.qrc">
     <normaloff>:/res/icons/dane_firmy.png</normaloff>:/res/icons/dane_firmy.png</iconset>
   </property>
   <property name="text">
    <string>Dane firmy</string>
   </property>
   <property name="iconText">
    <string>Firma</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="name" stdset="0">
    <string>plikDane_firmyAction</string>
   </property>
  </action>
  <action name="actionHelp_About">
   <property name="text">
    <string>O programie...</string>
   </property>
   <property name="iconText">
    <string>O programie...</string>
   </property>
   <property name="toolTip">
    <string>O programie...</string>
   </property>
   <property name="name" stdset="0">
    <string>pomocO_programieAction</string>
   </property>
  </action>
  <action name="actionProgram_Settings">
   <property name="icon">
    <iconset resource="../../res/qfaktury.qrc">
     <normaloff>:/res/icons/ustawienia.png</normaloff>:/res/icons/ustawienia.png</iconset>
   </property>
   <property name="text">
    <string>Ustawienia</string>
   </property>
   <property name="iconText">
    <string>Ustawienia</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+U</string>
   </property>
   <property name="name" stdset="0">
    <string>plikUstawieniaAction</string>
   </property>
  </action>
  <action name="actionHelp_Help">
   <property name="text">
    <string>Strona programu</string>
   </property>
   <property name="iconText">
    <string>Strona programu</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="name" stdset="0">
    <string>pomocPomocAction</string>
   </property>
  </action>
  <action name="actionCommodities_Edit">
   <property name="icon">
    <iconset resource="../../res/qfaktury.qrc">
     <normaloff>:/res/icons/edytuj_towar.png</normaloff>:/res/icons/edytuj_towar.png</iconset>
   </property>
   <property name="text">
    <string>Edycja</string>
   </property>
   <property name="iconText">
    <string>Edycja</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
   <property name="name" stdset="0">
    <string>towaryEdycjaAction</string>
   </property>
  </action>
  <action name="actionCommodities_Remove">
   <property name="icon">
    <iconset resource="../../res/qfaktury.qrc">
     <normaloff>:/res/icons/usun_towar.png</normaloff>:/res/icons/usun_towar.png</iconset>
   </property>
   <property name="text">
    <string>Usuń</string>
   </property>
   <property name="iconText">
    <string>Usuń</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
   <property name="name" stdset="0">
    <string>towaryUsunAction</string>
   </property>
  </action>
  <action name="actionCommodities_Add">
   <property name="icon">
    <iconset resource="../../res/qfaktury.qrc">
     <normaloff>:/res/icons/dodaj_towar.png</normaloff>:/res/icons/dodaj_towar.png</iconset>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
   <property name="iconText">
    <string>Towar/Usługa</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
   <property name="name" stdset="0">
    <string>towaryDodajAction</string>
   </property>
  </action>
  <action name="actionProgram_ExportToEDI">
   <property name="text">
    <string>Eksport do EDI</string>
   </property>
   <property name="iconText">
    <string>Eksport do EDI</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="name" stdset="0">
    <string>plikExportEDIAction</string>
   </property>
  </action>
  <action name="actionProgram_ExportToPDF">
   <property name="text">
    <string>Export do PDF</string>
   </property>
   <property name="iconText">
    <string>Export do PDF</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="name" stdset="0">
    <string>plikExport_pdfAction</string>
   </property>
  </action>
  <action name="actionExportToXML">
   <property name="text">
    <string>Export do XML</string>
   </property>
   <property name="iconText">
    <string>Export do XML</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="name" stdset="0">
    <string>plikExport_XMLAction</string>
   </property>
  </action>
  <action name="actionProgram_ImportEDI">
   <property name="text">
    <string>Import EDI</string>
   </property>
   <property name="iconText">
    <string>Import EDI</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="name" stdset="0">
    <string>plikImport_EDIAction</string>
   </property>
  </action>
  <action name="actionProgram_ImportXML">
   <property name="text">
    <string>Import XML</string>
   </property>
   <property name="iconText">
    <string>Import XML</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="name" stdset="0">
    <string>plikImport_XMLAction</string>
   </property>
  </action>
  <action name="actionInvoices_InvoiceCorrective">
   <property name="icon">
    <iconset resource="../../res/qfaktury.qrc">
     <normaloff>:/res/icons/nowa_korekta.png</normaloff>:/res/icons/nowa_korekta.png</iconset>
   </property>
   <property name="text">
    <string>Faktura korygująca</string>
   </property>
   <property name="iconText">
    <string>Nowa korekta</string>
   </property>
   <property name="shortcut">
    <string>K</string>
   </property>
   <property name="name" stdset="0">
    <string>fakturyKorektaAction</string>
   </property>
  </action>
  <action name="actionInvoices_InvoiceProForma">
   <property name="icon">
    <iconset resource="../../res/qfaktury.qrc">
     <normaloff>:/res/icons/nowa_proforma.png</normaloff>:/res/icons/nowa_proforma.png</iconset>
   </property>
   <property name="text">
    <string>Faktura Pro Forma</string>
   </property>
   <property name="iconText">
    <string>Pro forma</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
   <property name="name" stdset="0">
    <string>fakturyPFormaAction</string>
   </property>
  </action>
  <action name="actionHelp_BugReport">
   <property name="text">
    <string>Zgłoś błąd</string>
   </property>
  </action>
  <action name="actionInvoices_InvoiceDuplicate">
   <property name="icon">
    <iconset resource="../../res/qfaktury.qrc">
     <normaloff>:/res/icons/nowa_duplikat.png</normaloff>:/res/icons/nowa_duplikat.png</iconset>
   </property>
   <property name="text">
    <string>Duplikat faktury</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionInvoices_InvoiceGross">
   <property name="icon">
    <iconset resource="../../res/qfaktury.qrc">
     <normaloff>:/res/icons/nowa_brutto.png</normaloff>:/res/icons/nowa_brutto.png</iconset>
   </property>
   <property name="text">
    <string>Faktura Brutto</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionInvoices_Bill">
   <property name="icon">
    <iconset resource="../../res/qfaktury.qrc">
     <normaloff>:/res/icons/nowa_rachunek.png</normaloff>:/res/icons/nowa_rachunek.png</iconset>
   </property>
   <property name="text">
    <string>Rachunek</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionHelp_AboutQt">
   <property name="text">
    <string>O Qt...</string>
   </property>
   <property name="iconText">
    <string>O Qt...</string>
   </property>
   <property name="toolTip">
    <string>O Qt...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="4" margin="9"/>
 <tabstops>
  <tabstop>tabWidgetMain</tabstop>
  <tabstop>toolButtonApply</tabstop>
  <tabstop>tableViewInvoices</tabstop>
  <tabstop>dateEditFilterStart</tabstop>
  <tabstop>dateEditFilterEnd</tabstop>
  <tabstop>tableViewCounterparties</tabstop>
  <tabstop>tableViewCommodities</tabstop>
 </tabstops>
 <resources>
  <include location="../../res/qfaktury.qrc"/>
  <include location="../../res/qfaktury.qrc"/>
 </resources>
 <connections/>
</ui>
